src/controllers/revocation.ts

```diff
@@ -342,7 +331,7 @@ export class RevocationController {
 	 *         content:
 	 *           application/json:
 	 *             schema:
-	 *               $ref: '#/components/schemas/CredentialStatusResult'
+	 *               $ref: '#/components/schemas/CredentialStatusUpdateResponse'
```

Comment:
Similar story here, could we change this so that an `/update` returns _very_ similar to what `/create` does? Except the `created: true` will change to `updated: true`

Reply:
Done! ✅